thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -48,7 +55,17 @@ def output_to_zero(self):
                 pwm_msg.positive_width_us.append(neutral_pulse_width)
             self.pub_pwm.publish(pwm_msg)
 
+    def heartbeat_callback(self, msg):
+        self.prev_heartbeat = msg.stamp
+
     def callback(self, msg):
+        if (rospy.get_rostime() - self.prev_heartbeat).to_sec() > 1.0:
```

Comment:
We fear it a little bit. Maybe not enough to care now. It can be fixed by stamping the thruster_forces messages (maybe better maybe worse than a separate heartbeat from the allocator). Still think checking whatever heartbeats we use in a loop is better than checking in the callback, imo.